docs/sources/panels-visualizations/configure-data-links/index.md

```diff
@@ -26,35 +26,41 @@ weight: 80
 
 # Configure data links
 
-Data links allow you to provide more granular context to your links. You can create links that include the series name or even the value under the cursor. For example, if your visualization showed four servers, you could add a data link to one or two of them.
+Data links allow you to provide more granular context to your links. You can create links that include the series name or even the value under the cursor. For example, if your visualization shows four servers, you can add a data link to one or two of them. You can also link panels using data links.
 
-The link itself is accessible in different ways depending on the visualization. For the Graph you need to click on a data point or line, for a panel like
-Stat, Gauge, or Bar Gauge you can click anywhere on the visualization to open the context menu.
+The link itself is accessible in different ways depending on the visualization. For the time series visualization you need to click a data point or line:
 
-You can use data link variables or data links to create links between panels. You can use variables in data links to send people to a detailed dashboard with preserved data filters. For example, you could use variables to specify a time range, series, and variable selection. For more information, refer to [Data link variables](#data-link-variables).
+![Time series visualization with a data link displayed](screenshot-time-series-data-link-v10.3.png)
+
+For visualizations like stat, gauge, or bar gauge you can click anywhere on the visualization to open the context menu:
+
+![Stat visualization with a data link displayed](screenshot-stat-data-link-v10.3.png)
+
+If there's only one data link in the visualization, clicking anywhere on the visualization opens the link rather than the context menu.
 
 ## Data link variables
 
-You can use variables in data links to refer to series fields, labels, and values. For more information about data links, refer to [Data links](#data-links).
+Variables in data links let you send people to a detailed dashboard with preserved data filters. For example, you could use variables to specify a label, time range, series, or variable selection.
```

Comment:
**TO DATAVIZ REVIEWERS**: It would be great if we had an example here that shows how you format this. Please feel free to add one here.